mmv1/third_party/terraform/services/orgpolicy/resource_org_policy_policy_test.go

```diff
@@ -458,3 +458,108 @@ func testAccCheckOrgPolicyPolicyDestroyProducer(t *testing.T) func(s *terraform.
 		return nil
 	}
 }
+func TestAccOrgPolicyPolicy_EnforceParameterizedMCPolicy(t *testing.T) {
+        t.Parallel()    
+        
+        context := map[string]interface{}{
+                "org_id":        envvar.GetTestOrgFromEnv(t),
+                "random_suffix": acctest.RandString(t, 10),
+        }               
+
+        acctest.VcrTest(t, resource.TestCase{
+                PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+                ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+                CheckDestroy:             testAccCheckOrgPolicyPolicyDestroyProducer(t),
+                Steps: []resource.TestStep{
+                        {
+                                Config: testAccOrgPolicyPolicy_EnforceParameterizedMCPolicy(context),
```

Comment:
Could we add an update test step as well? You'd add another Config step with a different value for the field, then another ImportState step: https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test